server/auth/local/index.js

```diff
@@ -15,9 +18,19 @@ router.post('/', function(req, res, next) {
     if (!user) {
       return res.status(404).json({message: 'Something went wrong, please try again.'});
     }
+
     var profile = user.profile;
-    var token = signToken(user._id, user.role);
-    res.json({ token, profile });
+    User.findById(profile._id).then((user) => {
```

Comment:
This should not be part of this PR. Please branch from dev before working on separate feature in the future. Would not be able merge this due to problems in other PR